_scss/style.scss

```diff
@@ -0,0 +1,2 @@
+@import "./_reset";
+@import "./_main";
```

Comment:
`style.scss` will "include" stuff from _reset and _main in a single file. This is done by the node-sass compiler.